docs/quickstart.html

```diff
@@ -50,46 +50,47 @@ <h5>Kafka with KRaft</h5>
 
         <p>Kafka can be run using KRaft mode using local scripts and downloaded files or the docker image. Follow one of the sections below but not both to start the kafka server.</p>
 
-        <h5>Using downloaded files</h5>
+        <h6>Using downloaded files</h6>
 
         <p>Generate a Cluster UUID</p>
-        <pre><code class="language-bash">$ KAFKA_CLUSTER_ID="$(bin/kafka-storage.sh random-uuid)"</code></pre>
+        <pre><code class="language-bash">KAFKA_CLUSTER_ID="$(bin/kafka-storage.sh random-uuid)"</code></pre>
 
         <p>Format Log Directories</p>
-        <pre><code class="language-bash">$ bin/kafka-storage.sh format -t $KAFKA_CLUSTER_ID -c config/kraft/server.properties</code></pre>
+        <pre><code class="language-bash">bin/kafka-storage.sh format --standalone -t $KAFKA_CLUSTER_ID -c config/kraft/reconfig-server.properties</code></pre>
```

Comment:
It's more important to document `--initial-controllers` than `-standalone`. If only one way is going to be documented, it should certainly be `--initial-controllers`. (But I think documenting both is reasonable.)

Reply:
I document --initial-controllers in ops.html. I don't think we should use --initial-controllers in the quickstart for two reasons. It complicates the quick start because the user will have to generate another uuid for the controller's directory id instead of relying on the `kafka-storage` tool to safely generate the directory id.

As explain in another comment and in ops.html, the safest and most consistent way to bootstrap a cluster with many controllers is to use --standalone on one of the controller and `kafka-metadata-quorum add-controller` to add the rest of the controllers.

Reply:
ok